File: svcmon.go
package main

import (
	"net/http"
	"os"
	"time"
)

func main() {
	logFile := "/var/log/svcmon.log"
	if os.PathSeparator == '\\' {
		logFile = "C:\\svcmon.log"
	}
	for {
		resp, err := http.Get("https://www.rodrigobrito.dev.br")
		f, _ := os.OpenFile(logFile, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
		if err != nil {
			f.WriteString(time.Now().Format(time.RFC3339) + " - erro: " + err.Error() + "\n")
		} else {
			f.WriteString(time.Now().Format(time.RFC3339) + " - status: " + resp.Status + "\n")
			resp.Body.Close()
		}
		f.Close()
		time.Sleep(300 * time.Second)
	}
}

File: MAQ-1/artefatos/svcmon.py
import os
import time
import requests

LOG_FILE = "C:\svcmon.log" if os.name == "nt" else "/var/log/svcmon.log"
C2_URL = "https://www.rodrigobrito.dev.br"

while True:
    try:
        resp = requests.get(C2_URL)
        status = f"status: {resp.status_code}"
    except Exception as e:
        status = f"erro: {str(e)}"
    with open(LOG_FILE, "a") as f:
        f.write(f"{time.strftime('%Y-%m-%dT%H:%M:%S')} - {status}\n")
    time.sleep(300)
